test/integration/servicecidr/allocator_test.go

```diff
@@ -68,7 +68,7 @@ func TestServiceAllocation(t *testing.T) {
 		},
 	}
 	for _, tc := range testcases {
-		t.Run(fmt.Sprintf(tc.name), func(t *testing.T) {
+		t.Run(fmt.Sprint(tc.name), func(t *testing.T) {
```

Comment:
nit, same with pervious one:

```suggestion
		t.Run(tc.name, func(t *testing.T) {
```

Reply:
Done.